src/GRA.Controllers/MissionControl/PerformerManagementController.cs

```diff
@@ -241,6 +243,38 @@ public async Task<IActionResult> PerformerDelete(PerformerListViewModel model)
                 new { page = model.PaginateModel.CurrentPage });
         }
 
+        [HttpPost]
+        public async Task<IActionResult> PerformerSelectionDelete
+            (PerformerSelectionsViewModel model)
+        {
+            var settings = await _performerSchedulingService.GetSettingsAsync();
+            var schedulingStage = _performerSchedulingService.GetSchedulingStage(settings);
+            if (schedulingStage < PsSchedulingStage.SchedulingOpen)
+            {
+                return RedirectToAction(nameof(PerformerManagementController.PerformerSelections),
+                    new { id = model.Performer.Id });
+            }
+            try
+            {
+                var branchSelection = await _performerSchedulingService
+                    .GetBranchProgramSelectionByIdAsync(model.BranchSelectionId);
+                await _performerSchedulingService.DeleteBranchSelectionAsync(branchSelection);
+                var branch = await _siteService.GetBranchByIdAsync(branchSelection.BranchId);
+                var program = await _performerSchedulingService
+                    .GetProgramByIdAsync(branchSelection.ProgramId.Value);
+                branchSelection.IsDeleted = true;
+                ShowAlertSuccess($"{branch.Name}'s selection of \"{program.Title}\" is deleted!.");
+            }
+            catch (GraException gex)
+            {
+                ShowAlertDanger("Unable to delete selection: ", gex);
+                _logger.LogError("Error deleting branch selection Id" +
+                    $"{model.BranchSelectionId} by user {GetId(ClaimType.UserId)}: {gex}", gex);
```

Comment:
LogError should use arguments to set the variables instead of string interpolation
Example: 

    _logger.LogError(gex,
        "Problem displaying index for user {GetActiveUserId}: {Message}",
        GetActiveUserId(),
        gex.Message);

Reply:
Fixed in 9bf3f38be7dda584a234205340b36076a97d5cc0